dists/fs-repo-0-to-1/current

```diff
@@ -0,0 +1 @@
+v2.0.0
```

Comment:
Why are these all v2?

Reply:
Previously, these did not have their own versions, so they inherited the version of the fs-repo-migrations repo.  Since we are going to version the fs-repo-migration to 2.0.0 for next release, I thought that we would want the individual migrations versioned likewise.  We can choose any versioning we want though, nothing depends on knowing their version.